src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/MessageProcessorFunctionalTest.java

```diff
@@ -373,8 +376,12 @@ public void should_not_process_dlq_message_unless_other_messages_exist_with_same
                     () -> {
                         final EventMessageQueryResponse messagesInReadyState = getMessagesFromDb(MessageState.READY);
                         if (messagesInReadyState != null) {
-                            assertEquals(1, messagesInReadyState.getNumberOfMessagesFound());
-                            assertEquals(caseId, messagesInReadyState.getCaseEventMessages().get(0).getCaseId());
+
+                            List<CaseEventMessage> returnedCase = messagesInReadyState.getCaseEventMessages().stream()
```

Comment:
I don't fully understand what this test is trying to test.  
Seems to put a message in the DLQ and then checking it is in a `Ready` state?

Unless I've missed something there's nothing to stop that message moving from `Ready` to `Processed` between the time you put it in the DLQ and check its state.  Equally it may not have got to `Ready` **yet.**

Reply:
The test cannot fully control when a DLQ message gets inserted into the DB or gets its status updated.  It is therefore relying on behaviour that is not guaranteed.

Reply:
This test ignored until the rest of the test suite is stable.